c++/src/capnp/any.h

```diff
@@ -806,6 +806,31 @@ inline Orphan<AnyPointer> Orphan<AnyPointer>::releaseAs() {
   return kj::mv(*this);
 }
 
+enum class StructEqualityResult {
+  NOT_EQUAL,
+  EQUAL,
+  UNKNOWN_CONTAINS_CAPS
+};
+
+inline kj::StringPtr KJ_STRINGIFY(StructEqualityResult res) {
```

Comment:
Probably true.  I figured it would be fairly unusual to need this linked in, so putting it in the header gives the compiler a better chance of eliminating it as dead code.  Right now, it's only needed for the tests.  I went ahead and removed it.